common_utils/utils.go

```diff
@@ -148,10 +149,23 @@ type InputDataBlock struct {
 	Finalized            bool `json:"finalized"`
 }
 
-func ConvertBytesToStrings(b [][32]byte) []string {
+func ConvertBytesToStrings(b interface{}) []string {
 	var s []string
-	for _, v := range b {
-		s = append(s, "0x"+hex.EncodeToString(v[:]))
+	switch v := b.(type) {
+	case [][]byte:
```

Comment:
It looks like using type merging can simplify the original code.

> case [][]byte, [][32]byte, common.Proof:
    for _, bytes := range v {
        s = append(s, "0x"+hex.EncodeToString(bytes[:]))
    }

Reply:
I tried, but it reports error for common.Proof